mmv1/products/notebooks/api.yaml

```diff
@@ -841,12 +850,16 @@ objects:
         description: |
           The config settings for accessing runtime.
         properties:
-          - !ruby/object:Api::Type::String
+          - !ruby/object:Api::Type::Enum
```

Comment:
Why update this from string to enum? We have been tending to do the opposite so we don't have to keep up with the API on what's actually validated. 
I'd be very hesitant to approve String -> Enum without strong reasoning.

Reply:
i can revert but plan time validations are helpful when API (more often than not...) is vague in its response for bad input. lmk what you decide

Reply:
That's understandable. We've run into a fair amount of complaints when the validation is over-restrictive, mostly due to the API enabling new values and the provider being too slow in updating.
Removing the helpfulness of plan-time validations in exchange for reduction of future maintenance is the direction our team has been heading, so I'll go with with reverting at this time.

Reply:
reverted